src/components/Dashboard/StaffNotation.vue

```diff
@@ -22,7 +22,7 @@
             filingType="courtOrder"
             courtOrderNumberRequired="true"
         />
-        <div class="filing-item__actions">
+        <div class="filing-item__actions mb-2">
```

Comment:
It's usually better to add "top margin" to the element below this one, instead of adding "bottom margin" to this one. (Principle: it's easier to know what's above you than what's below you.)

What are you really trying to do here -- align this menu within the header?

![image](https://user-images.githubusercontent.com/10002889/119575537-1288b980-bd6c-11eb-85f7-0fcc311f1de2.png)

It might work better to use negative margin at the top of this div to "pull it up" and aligned with the h2 in the header.

Reply:
>What are you really trying to do here -- align this menu within the header?

Yes.

>It might work better to use negative margin at the top of this div to "pull it up" and align it vertically with the h2 in the header.

The H2 in the header has bottom margin. I'm trying the negative margin/padding with no success. Let me see...

Reply:
It worked applying the negative margin to the main div. Thanks.